### PR TITLE
Adding an initial README and list

### PR DESCRIPTION
**Pull request by arschles**
This PR adds an initial README, contribution guide, and list

### CODE REVIEW

File: THELIST.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# The List of Brokers
```
Comment (carolynvs) [inline THELIST.md]: What about putting it on the README so that it's the first thing you see?

Comment (arschles) [reply THELIST.md]: I want to make sure the first thing people see is some context, the code of conduct, etc... I've moved the "want to jump right to the list?" part up. Thoughts?

Comment (carolynvs) [reply THELIST.md]: How about at the bottom of the readme? 😇

Comment (arschles) [reply THELIST.md]: fair nuff! will do

**Review by carolynvs — APPROVED** [review]

**Comment by carolynvs** [comment]
Thank you for bowing to my tyrannical demands! 👍

**Pull request merged**